src/routes/controllers/productsControllers/createProduct.js

```diff
@@ -3,67 +3,68 @@ import validateData from '../../../helpers/validateData';
 import ProductModel from '../../../models/product';
 
 const imageValidation = Joi.object({
-  name: Joi.string(),
-  src: Joi.string(),
+	altName: Joi.string(),
+	name: Joi.string(),
+	src: Joi.string(),
```

Comment:
Is the src a web url or a path from the directory? The reason i'm asking since i saw a code snippet where the src was path from the directory.

Reply:
its the endpoint path without the proxy so it will be '/products/images/:filename' and then on frontend when you call the endpoint to get img it will be `<Image src={REACT_APP_PROXY + image.src} />`